src/services/shortLink/test/index.test.ts

```diff
@@ -0,0 +1,552 @@
+      });
+      it('Forbidden if patch short links with write permission', async () => {
+        const { item } = await mockPostData({
+          author: actor,
+          member: bob,
+          permission: PermissionLevel.Write,
+        });
+        const post = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+
+        // sign in as bob
+        ({ app } = await build({ member: bob }));
+
+        const response = await injectPatch(app, MOCK_ALIAS, { platform: GraaspPlatform.BUILDER });
+        expect(response.statusCode).toEqual(StatusCodes.FORBIDDEN);
+      });
+    });
+
+    describe('Success', () => {
+      it('Success if patch short links with admin permission', async () => {
+        const { item } = await mockPostData({
+          author: actor,
+          member: bob,
+          permission: PermissionLevel.Admin,
+        });
+        const post = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+
+        // sign in as bob
+        ({ app } = await build({ member: bob }));
+
+        const response = await injectPatch(app, MOCK_ALIAS, { platform: GraaspPlatform.BUILDER });
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().platform).toEqual(GraaspPlatform.BUILDER);
+      });
+      it('Success if patch short links for the creator', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const post = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+
+        const response = await injectPatch(app, MOCK_ALIAS, { platform: GraaspPlatform.BUILDER });
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().platform).toEqual(GraaspPlatform.BUILDER);
+      });
+
+      it('Success if patch full short links for the creator', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const post = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+
+        const response = await injectPatch(app, MOCK_ALIAS, { platform: GraaspPlatform.BUILDER });
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().platform).toEqual(GraaspPlatform.BUILDER);
+      });
+    });
+
+    describe('Conflict and bad request', () => {
+      it('Conflict if patch short links alias with existing one', async () => {
+        const { item } = await mockPostData({ author: actor });
+        let post = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+        post = await injectPost(
+          app,
+          mockShortLinkPayload({ item_id: item.id, alias: MOCK_ALIAS + 2 }),
```

Comment:
Prefer string templates in this case
```suggestion
          mockShortLinkPayload({ item_id: item.id, alias: `${MOCK_ALIAS}2` }),
```